packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -338,7 +338,19 @@ class OptimisticOracleContractMonitor {
   }
 
   _generateUILink(transactionHash, chainId) {
-    return `<${this.optimisticOracleUIBaseUrl}/request?transactionHash=${transactionHash}&chainId=${chainId}|View in UI>`;
```

Comment:
note this change: the OO UI now needs different URLS based on the OO type. this switch case supports this.